lib/iris/experimental/ugrid.py

```diff
@@ -2769,8 +2773,7 @@ def __init__(
 
         # Get the 'coord identity' metadata from the relevant node-coordinate.
         # N.B. mesh.coord returns a dict
-        node_coords = self.mesh.coord(include_nodes=True, axis=self.axis)
-        (node_coord,) = list(node_coords.values())
+        node_coord = self.mesh.coord(include_nodes=True, axis=self.axis)
```

Comment:
@pp-mo Oh blessed simplicity... that is way, way, way better IMHO

Reply:
nice ! :+1: